### DISCUSSION TITLE
Allow choosing the UWB engine path as desired

### DISCUSSION THREAD

**Discussion by pavelsimo** [Voltstro-Studios/UnityWebBrowser#315] [Requests]
Hi UWB-team,

First of all, thanks for the great package! it works as advertised :)

I have a use case where I would like to choose the UWB engine path. This seems to be possible only when running in the editor, but not for a build.

I'm talking about function below, it is currently default to `Application.dataPath`:

Would it be possible to enable a setting for changing this path? perhaps something similar to `webBrowser.browserClient.LogPath` and `webBrowser.browserClient.CachePath`.

```csharp
        /// <summary>
        ///     Gets the folder that the UWB process application lives in
        /// </summary>
        /// <returns></returns>
        public static string GetBrowserEnginePath(Engine engine)
        {
            //Editor
#if UNITY_EDITOR
            return EngineManager.GetEngineDirectory(engine);

            //Player builds (Standalone)
#elif UNITY_STANDALONE_OSX
            return Path.GetFullPath($"{Application.dataPath}/Resources/Data/UWB/");
#elif UNITY_STANDALONE
		    return Path.GetFullPath($"{Application.dataPath}/UWB/");
#else       //Unsupported platform, UWB shouldn't run anyway
            return null;
#endif
        }
```

Thanks,
Pavel

**Comment by Voltstro** [comment]
I guess this could be added as a `webBrowser.browserClient.RuntimePath` property.

Only concern is with the build post processor. If each UWB instance has a different path, the post processor will have no idea where to put the files. But this feature can just be offered as an advanced feature, and the management of the engine files can be done by you.
